docs/resources/post_process_aws_connection.md

```diff
@@ -0,0 +1,60 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "clumio_post_process_aws_connection Resource - terraform-provider-clumio"
+subcategory: ""
+description: |-
+  Post process Clumio AWS Connection Resource used to post-process AWS connection to Clumio.
+---
+
+# clumio_post_process_aws_connection (Resource)
+
+Post process Clumio AWS Connection Resource used to post-process AWS connection to Clumio.
```

Comment:
Nit: `Clumio Post-Process AWS Connection Resource used to denote that the Clumio AWS Connection Resource with the same token is now ready for use`?